src/_components/Footer.jsx

```diff
@@ -84,24 +87,26 @@ export default function Footer() {
                 </div>
               </div>
             </div>
-            <div className="grid grid-cols-2 gap-20 sm:gap-60 sm:grid-cols-3 ">
+            <div className="grid grid-cols-2 gap-20 sm:grid-cols-4">
               <div>
-                <h2 className="mb-6  text-md font-semibold font- poppins text-white uppercase ">
+                <h2 className="mb-6 text-md font-semibold font- poppins text-white uppercase ">
                   General
                 </h2>
                 <ul className="text-gray-300  font-medium">
                   <li className="mb-4 font- poppins">
```

Comment:
Since these have the same styles you could consider using the map function in javascript to reduce the code:

```
const links = [
   {text: "MISSION", url: "/mission"},
   {text: "PROJECTS", url: "/projects"}
   # AND SO ON
 }

links.map((link) => {
    <li className="mb-4 font- poppins">
           <a href={link.url} className="hover:underline">{link.text}</a>
     </li>
  )
})
```

I did something similar on the `src/_components/Home/Hero.jsx` after Miguel recommended, just help remove redundant styles